src/insert-extension.js

```diff
@@ -34,14 +34,25 @@ export function resetExtensionScriptInsertedOrPresentForTesting() {
 /**
  * Check script info in HTML head and make update if necessary
  * @param {!Window} win
- * @param {!Element} element
  * @param {string} extension
+ * @param {boolean} opt_ignoreElementExistenceCheck if true, will not check and
+ *    require that extension element exist in document
  */
-export function insertAmpExtensionScript(win, element, extension) {
+export function insertAmpExtensionScript(
+    win, extension, opt_ignoreElementExistenceCheck) {
   if (extension == 'amp-embed') {
     extension = 'amp-ad';
   }
-  if (isAmpExtensionScriptRequired(win, element, extension)) {
+  if (!opt_ignoreElementExistenceCheck) {
```

Comment:
Yeah, we inherited this check from (IIRC) @zhouyx , and we just tried to preserve it as much as possible.  I'm happy to kill it if we think it isn't serving much purpose globally.  Yuxuan: any opinions?